pysradb/cli.py

```diff
@@ -34,15 +38,23 @@ def error(self, message):
 
 def _print_save_df(df, saveto=None):
     if saveto:
-        df.to_csv(saveto, index=False, header=True, sep="\t")
+        if saveto.split(".")[-1].strip().lower() == "csv":
+            #  if the save file format is csv
+            df.to_csv(saveto, index=False, header=True)
+        else:
+            df.to_csv(saveto, index=False, header=True, sep="\t")
     else:
         if len(df.index):
             pd.set_option("display.max_colwidth", None)
             # Bug in pandas 0.25.3: https://github.com/pandas-dev/pandas/issues/24980
             # causes extra leading spaces
-            to_print = df.to_string(index=False, justify="left", col_space=0).lstrip()
+            to_print = df.to_string(
+                index=False, justify="left", header=False, col_space=0
+            ).lstrip()
             to_print_split = to_print.split(os.linesep)
-            to_print = []
+            # Header formatting seems off when it is added via to_string()
```

Comment:
I think this problem might be difficult to understand and check if this is still a problem in the future. If you have a case that doesn't work it might be worth writing a small unit test for `_print_save_df`.